source/_docs/cms-api.md

```diff
@@ -0,0 +1,173 @@
+
+
+## Deploying your CMS API site in Pantheon
+
+There are no differences when deploying a CMS API than a normal site. You may follow our [Launch Essentials](/docs/guides/launch/) guide to get started, and also refer to the [Pantheon Workflow](/docs/pantheon-workflow) page for how to deploy your code or manage your content.
+
+Make sure to check the [Best Practice Checklist for Going Live](/docs/guides/launch/next-steps/) and activate other features like [New Relic](/docs/new-relic/) for monitoring, or enabling [Solr](/docs/solr) and [Redis](/docs/redis/)
+
+You can also write scripts in [Guzzle](http://docs.guzzlephp.org/en/latest/), which is a great tool (and included in Drupal 8 Core) for testing. You can also use cURL via the command line or PHP.
+
+### Pantheon Global CDN
+
+All sites will benefit from the use of [Global CDN](/global-cdn/), a core platform offering for improved performance and security. It enables an advanced caching mechanism, and have free and automated HTTPS by default.
+
+To learn more, read our [Global CDN documentation](/docs/global-cdn/).
+
+<!-- How does Global CDN cache REST API resources, if there are any differences related to Varnish config, etc -->
+
+<!-- ### Performance Monitoring -->
+
+<!-- With New Relic enabled, page requests appear as transactions and not sessions for REST API resources retrieved.
+The best way to monitor its performance is to remember the most common transactions, hooks or name patterns that appear in the logs. -->
+
+<!-- New Relic REST API patterns here -->
+
+<!-- ### Redis and Object Caching -->
+
+<!-- tips on how Object Cache is best used for REST API CMS -->
+
+<!-- ### Solr Search with API -->
+
+<!-- example cases for using Solr Search with CMS API -->
+
+
+## Frequently Asked Questions
+
+### How can I troubleshoot my CMS API?
+
+To test or troubleshoot your REST API means to test the `GET`, `POST` or `DELETE` methods through HTTP. The most commonly used tools are the following:
+
+* [Postman](https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop?hl=en)
+* [Dev HTTP Client](https://chrome.google.com/webstore/detail/dev-http-client/aejoelaoggembcahagimdiliamlcdmfm/related)
+* [Restlet Client](https://chrome.google.com/webstore/detail/restlet-client-rest-api-t/aejoelaoggembcahagimdiliamlcdmfm)
+
+Note: These are Chrome Plugin tools, but there are similar tools for other browsers
+
+### Can I place a single page app in my root directory?
+
+Yes, but instead of using `index.html`, we use `override.html` as the default page to be loaded. It is also possible to have a sub-directory that contains singe-page apps with HTML, JS or CSS files. For deployment, these are treated the same way as other static assets in the platform.
+
+
+## See Also
```

Comment:
@eabquina the links below don't point to anything currently on the site.

Reply:
@alexfornuto  I just placed the related blog entries from Pantheon, and references I used for making the docs page.

Should they be removed? Or should I format as external links?